apps/api/src/database/repositories/boost.repository.ts

```diff
@@ -536,7 +542,7 @@ class BoostRepository {
   async userBoosts(
     { userId, competitionId }: { userId: string; competitionId: string },
     tx?: Transaction,
-  ) {
+  ): Promise<Record<string, bigint>> {
```

Comment:
This is fine, but just wonder why you don't let the return type be inferred?

Reply:
I just think explicit types are generally better than implicit ones